src/Compute/Compute/help/Export-AzLogAnalyticRequestRateByInterval.md

```diff
@@ -172,6 +172,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -NoWait
+Returns immediately with status of request
```

Comment:
Starts the operation and returns immediately, before the operation is completed.  The user will need to use some other mechanism to determine when the operation completes.

Reply:
Apply this to all instances